x/contracts/test/util.go

```diff
@@ -12,7 +12,7 @@ import (
 )
 
 func CompileTest(contractName string) error {
-	cmd := exec.Command("cargo", "build", "-p", contractName, "--release", "--target=wasm32-unknown-unknown", "--target-dir=./target")
+	cmd := exec.Command("cargo", "build", "-p", contractName, "--release", "--target-dir=./target")
```

Comment:
Why did you remove --target?

Reply:
if i included the --target then our build script wouldn't activate and thus not properly create the `.wasm` file for the examples